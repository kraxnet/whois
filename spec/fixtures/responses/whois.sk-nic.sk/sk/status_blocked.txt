% For more information on Whois status codes, please visit https://icann.org/epp
%
% This WHOIS service is provided by SK-NIC and only contains
% information pertaining to Internet domain names registered by our
% our customers. By using this service you are agreeing (1) not to use any
% information presented here for any purpose other than determining
% ownership of domain names, (2) not to store or reproduce this data in
% any way without SK-NIC's approval, (3) not to use any high-volume, automated, electronic processes
% to obtain data from this service. Abuse of this service is monitored and
% actions in contravention of these terms will result in being permanently
% blacklisted.
%
% Access to the whois service is rate limited.

Domain:                       3dmodely.sk
Registrant:                   SHOP-0007
Admin Contact:                SHOP-0007
Tech Contact:                 SHOP-0007
Registrar:                    SHOP-0007
Created:                      2017-03-15
Updated:                      2017-09-06
Valid Until:                  2018-03-15
Nameserver:                   ns1.sk-nik.sk
Nameserver:                   ns2.sk-nik.sk
EPP Status:                   clientRenewProhibited, clientTransferProhibited, clientUpdateProhibited, clientDeleteProhibited

Registrar:                    SHOP-0007
Organization:                 Shop, s.r.o.
Organization ID:              36725455
Phone:                        +421.905309525
Email:                        hulla@look4.sk
Street:                       Popolná 9603/49
City:                         Bratislava-Rača
Postal Code:                  831 06
Country Code:                 SK
Created:                      2017-09-01
Updated:                      2017-09-10
