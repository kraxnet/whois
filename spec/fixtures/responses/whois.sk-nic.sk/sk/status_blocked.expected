#status
  %s == :registered

#available?
  %s == false

#response_throttled?
  %s == false

#registered?
  %s == true

#domain
  %s == "3dmodely.sk"
